seatunnel-e2e/seatunnel-connector-v2-e2e/connector-iceberg-e2e/src/test/resources/iceberg/filter_iceberg_source.conf

```diff
@@ -0,0 +1,108 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+######
+###### This config file is a demonstration of streaming processing in seatunnel config
+######
+
+env {
+  parallelism = 2
+  job.mode = "BATCH"
+}
+
+source {
+  Iceberg {
+    schema {
+      fields {
+        f2 = "boolean"
+        f1 = "bigint"
+        f3 = "int"
+        f4 = "bigint"
+        f5 = "float"
+        f6 = "double"
+        f7 = "date"
+        f9 = "timestamp"
+        f10 = "timestamp"
+        f11 = "string"
+        f12 = "bytes"
+        f13 = "bytes"
+        f14 = "decimal(19,9)"
+        f15 = "array<int>"
+        f16 = "map<string, int>"
+      }
+    }
+    catalog_name = "seatunnel"
+    iceberg.catalog.config={
+      "type"="hadoop"
+      "warehouse"="file:///tmp/seatunnel/iceberg/hadoop/"
+    }
+    namespace = "database1"
+    table = "source"
+    plugin_output = "iceberg"
+    query = "select f1, f2 from t where f1 = 10"
+  }
+}
+
+transform {
+}
+
+sink {
+  Assert {
+    plugin_input = "iceberg"
+    rules =
+      {
+      row_rules = [
+                  {
+                    rule_type = MAX_ROW
+                    rule_value = 1
+                  }
+                   {
+                    rule_type = MIN_ROW
+                    rule_value = 1
+                  }
+                ]
+        field_rules = [
+          {
+            field_name = f1
+            field_type = bigint
+            field_value = [
+              {
+                rule_type = NOT_NULL
+              },
+              {
+                rule_type = MIN
+                rule_value = 10
+              },
+              {
+                rule_type = MAX
+                rule_value = 10
+              }
```

Comment:
```suggestion
```

field_rules doesn't support this.

Reply:
actually i'm referring to the https://github.com/apache/seatunnel/blob/ff5fc129b84fee42e0fa488f27ff2254d37ca7a2/docs/en/connector-v2/sink/Assert.md?plain=1#L160 , the doc example is using the below config in the field_value ,
```
{
              rule_type = MIN
              rule_value = 32767
            },
            {
              rule_type = MAX
              rule_value = 2147483647
            }
``` 

Please correct me if anything wrong.

Reply:
to make the config file more shorter, i change it to use the `equals_to`